modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/components/columns/OptionsColumn/OptionsColumn.jsx

```diff
@@ -36,16 +45,26 @@ const OptionsColumn = ({
 			saveDisabled={saveDisabled}
 		/>
 	) : (
-		<ButtonDropDown
-			customDropDownButton={
-				<ButtonWithIcon displayType="null" small symbol="ellipsis-v" />
-			}
-			items={userOptions}
-			menuElementAttrs={{
-				className: 'p-0',
-			}}
-			menuWidth="shrink"
-		/>
+		<>
+			<ClayTooltipProvider>
+				<span>
+					<ButtonDropDown
+						customDropDownButton={
+							<ButtonWithIcon
+								displayType="null"
+								small
+								symbol="ellipsis-v"
+							/>
+						}
+						items={userOptions}
+						menuElementAttrs={{
+							className: 'p-0',
+						}}
+						menuWidth="shrink"
+					/>
+				</span>
+			</ClayTooltipProvider>
+		</>
```

Comment:
If the fragment makes no difference here, remove plz

```suggestion
		<ClayTooltipProvider>
			<span>
				<ButtonDropDown
					customDropDownButton={
						<ButtonWithIcon
							displayType="null"
							small
							symbol="ellipsis-v"
						/>
					}
					items={userOptions}
					menuElementAttrs={{
						className: 'p-0',
					}}
					menuWidth="shrink"
				/>
			</span>
		</ClayTooltipProvider>
```